src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoNumberGroupSizes.cs

```diff
@@ -67,21 +67,15 @@ public void NegTest3()
 
         // TestNumberGroupSizesLocale: Verify value of property NumberGroupSizes for specific locales
         [Theory]
-        [InlineData("en-US", 3, 0)]
-        [InlineData("ur-IN", 3, 2)]
-        [ActiveIssue(3243)]
-        public void TestNumberGroupSizesLocale(string locale, int primaryGroupSize, int secondaryGroupSize)
+        [InlineData("en-US")]
+        [InlineData("ur-IN")]
+        public void TestNumberGroupSizesLocale(string locale)
```

Comment:
I liked this better the other way, since it showed at a glance why these two cultures are both good test cases.